src/main/java/org/myrobotlab/kinematics/PoseSequence.java

```diff
@@ -0,0 +1,27 @@
+package org.myrobotlab.kinematics;
+
+public class PoseSequence {
```

Comment:
Rename this class to something easier to understand  like a pose step.. or just a step  (as a base class)..  ..   "sequence" makes me think this is a list of steps. but it's not, it's just one of them.